pusher/src/Controller/AuthenticateController.ts

```diff
@@ -178,7 +179,7 @@ export class AuthenticateController extends BaseHttpController {
                             if (!code && !nonce) {
                                 return res.json({ ...resUserData, authToken: token });
                             }
-                            console.error("Token cannot to be check on OpenId provider");
+                            console.error("Token cannot to be checked on OpenId provider");
```

Comment:
```suggestion
                            console.error("Token cannot be checked on OpenId provider");
```

Reply:
And the message below should be:

```
User cannot be connected to OpenId provider
```

D'ailleurs, est-ce que ça ne serait pas plutôt:

```
User cannot be identified by OpenId provider
```